src/eth/TransactionObject.js

```diff
@@ -44,9 +45,12 @@ export default class TransactionObject extends TransactionLifeCycle {
     return this._returnValue();
   }
 
-  async confirm(count = 3) {
-    await this.mine();
+  async confirm(delay = this._confirmedBlockCount) {
+    let count;
+    typeof delay === 'number' ? (count = delay) : (count = 5);
```

Comment:
It's necessary since `confirmedBlockCount` is an optional config value, so it may be undefined. Basically this logic makes it so the first priority is an explicit parameter, the second priority is the global config option, and the third priority is the default (5).

Reply:
I think it's clearer if you have the default value where you set the value in Web3Service (see [this comment](https://github.com/makerdao/dai.js/pull/41#discussion_r220373661))

Reply:
I pushed an unrelated change to that line after you reviewed this, so I missed the other comment. That definitely works